pkg/apis/extensions/types.go

```diff
@@ -185,6 +185,7 @@ type ThirdPartyResourceData struct {
 	Data []byte `json:"name,omitempty"`
 }
 
+// +genclient=true
```

Comment:
Can you please move this one to the next PR (where you really generate the code)?